src/Microsoft.SqlTools.ServiceLayer/Scripting/Contracts/ScriptingRequest.cs

```diff
@@ -5,6 +5,7 @@
 
 using System.Collections.Generic;
 using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Connection.Contracts;
 
```

Comment:
why is this new using required in this file?  I don't see any changes in the PR that appear to necessitate this.